src/Slices/ServiceInventory/UI/Tabs/Tabs.tsx

```diff
@@ -128,3 +178,80 @@ const configTab = (
   isDisabled,
   ref,
 });
+
+/**
+ * Creates an array of documentation tab descriptors.
+ *
+ * @param row - The row object.
+ * @param service - The service model.
+ * @returns An array of documentation tab descriptors.
+ */
+const documentationTab = (
```

Comment:
cosmetic things, but it should be documentationTabs

Reply:
updated